libImaging/Quant.c

```diff
@@ -1503,8 +1503,10 @@ ImagingQuantize(Imaging im, int colors, int mode, int kmeans)
            should be done by a simple copy... */
 
         for (i = y = 0; y < im->ysize; y++)
-            for (x = 0; x < im->xsize; x++, i++)
+            for (x = 0; x < im->xsize; x++, i++) {
                 p[i].c.r = p[i].c.g = p[i].c.b = im->image8[y][x];
+                p[i].c.a = 255;
```

Comment:
Without this, `libimagequant` acts very weird. Come to think of it, not having this line is actually a bug:

- Pillow supports 32-bit palettes
- We run this method with an image using a 32-bit palette
- We send this image to a quantizer that supports such 32-bit palettes
- ...it receives random garbage in the alpha channel